corehq/apps/commtrack/tests/util.py

```diff
@@ -164,6 +164,7 @@ def setUp(self):
 
         self.loc = make_loc('loc1')
         self.sp = make_supply_point(self.domain.name, self.loc)
+        self.loc.save()
```

Comment:
the signal that links the stock state created by make_supply_point to the location in the sql database is only fired when the location is saved. when i changed the manager for the stockstate model to exclude stock states with no locations, i needed to get ensure they would be properly linked and saving the location again was the only way to do that.